samples/snippets/fsharp/tour.fs

```diff
@@ -1,468 +1,668 @@
-// The following is a modified version of the F# Tutorial script that comes in F# templates for Visual Studio.
-
-
-// ---------------------------------------------------------------
-//         Integers and basic functions
-// ---------------------------------------------------------------
-
-module Integers =
-
+// This sample will guide you through elements of the F# language.
+//
+// *******************************************************************************************************
+//   To execute the code in F# Interactive, highlight a section of code and press Alt-Enter or right-click 
+//   and select "Execute in Interactive".  You can open the F# Interactive Window from the "View" menu. 
+// *******************************************************************************************************
+//
+// For more about F#, see:
+//     http://fsharp.org
+//     https://docs.microsoft.com/en-us/dotnet/articles/fsharp/
+//
+// To see this tutorial in documentation form, see:
+//     https://docs.microsoft.com/en-us/dotnet/articles/fsharp/tour
+//
+// To learn more about applied F# programming, use
+//     http://fsharp.org/guides/enterprise/
+//     http://fsharp.org/guides/cloud/
+//     http://fsharp.org/guides/web/
+//     http://fsharp.org/guides/data-science/
+//
+// To install the Visual F# Power Tools, use
+//     'Tools' --> 'Extensions and Updates' --> `Online` and search
+//
+// For additional templates to use with F#, see the 'Online Templates' in Visual Studio, 
+//     'New Project' --> 'Online Templates'
+
+// F# supports three kinds of comments:
+
+//  1. Double-slash comments.  These are used in most situations.
+(*  2. ML-style Block comments.  These aren't used that often. *)
+/// 3. Triple-slash comments.  These are used for documenting functions, types, and so on.
+///    They will appear as text when you hover over something which is decorated with these comments.
+///
+///    They also support .NET-style XML comments, which allow you to generate reference documentation,
+///    and they also allow editors (such as Visual Studio) to extract information from them.
+///    To learn more, see: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/xml-documentation
+
+
+// Open namespaces using the 'open' keyword.
+//
+// To learn more, see: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/import-declarations-the-open-keyword
+open System
+
+
+/// Modules are the primary way to organize functions and values in F#. This module contains some
+/// basic values involving basic numeric values computed in a few different ways.
+///
+/// To learn more, see: https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/modules
+module IntegersAndNumbers = 
+
+    /// This is a sample integer.
```

Comment:
These are the triple-slash comments that give you QuickInfo summaries on hover:

![hover-doc](https://cloud.githubusercontent.com/assets/6309070/21702609/5494d2a4-d361-11e6-8d2c-dc277d0a66e9.PNG)

It's a lot more common to elide the `<summary>` tag in F#, since the language service assumes anything outside of a tag is just part of the summary.